openassessment/templates/legacy/edit/oa_edit_basic_settings_list.html

```diff
@@ -162,4 +162,30 @@
             {% trans "Specify whether learners can see the rubric while they are working on their response." %}
         </p>
     </li>
+    <li id="openassessment_allow_learner_resubmissions_wrapper" class="field comp-setting-entry">
+        <div class="wrapper-comp-setting">
+            <label for="openassessment_allow_learner_resubmissions_selector" class="setting-label">{% trans "Allow Learner Resubmissions"%}</label>
+            <select id="openassessment_allow_learner_resubmissions_selector" class="input setting-input" name="alow learner resubmissions">
+                <option value="0">{% trans "False"%}</option>
+                <option value="1" {% if allow_learner_resubmissions %} selected="true" {% endif %}>{% trans "True"%}</option>
+            </select>
+        </div>
+        <p class="setting-help">
+            {% trans "Set to TRUE to allow learners to resubmit ungraded work. Set to FALSE if resubmissions are not allowed." %}
+        </p>
+        <div id="openassessment_resubmissions_grace_period_wrapper" class="{% if not allow_learner_resubmissions %}is--hidden{% endif %}">
+            <div class="wrapper-comp-setting">
+                <label for="openassessment_resubmissions_grace_period" class="setting-label">{% trans "Resubmission Grace Period" %}</label>
+                <input
+                    type="text"
+                    id="openassessment_resubmissions_grace_period"
+                    class="input setting-input"
+                    placeholder="dd:hh:mm"
+                    value="{{ resubmissions_grace_period }}"
+                >
+            </div>
+            <p class="setting-help">{% trans "Specify the maximum period during which resubmissions are allowed. If left blank, learners will be able to reset their submission as long as it has not been graded." %}</p>
```

Comment:
When does this grace period start? It's not clear from this text.

Reply:
I updated the text. Do you think it is clearer?

Done: https://github.com/openedx/edx-ora2/commit/7c8e4d973412037d554b3845231c171f99e1674e